robottelo/hosts.py

```diff
@@ -1035,6 +1035,12 @@ def is_upstream(self):
     def url(self):
         return f'https://{self.hostname}'
 
+    @cached_property
+    def url_katello_ca_rpm(self):
+        """Return the Katello cert RPM URL"""
+        pub_url = urlunsplit(('http', self.hostname, 'pub/', '', ''))  # use url with https?
+        return urljoin(pub_url, 'katello-ca-consumer-latest.noarch.rpm')
+
```

Comment:
Moved from `Satellite` to `Capsule` class, so that we can register a contenthost to both, Satellite and Capsule.